lib/FIX/Lite.pm

```diff
@@ -2,7 +2,7 @@ package FIX::Lite;
 
 use vars qw($VERSION @ISA);
 use warnings;
-use strict;
+#use strict;
```

Comment:
I commented it out temporarily yesterday while debugging. I have
uncommented it back.

On Wed, Dec 16, 2015 at 4:07 PM, Jean-Yves Sireau <notifications@github.com>
wrote:

> In lib/FIX/Lite.pm
> <https://github.com/agapoff/FIX-Lite/pull/1#discussion_r47746105>:
>
> > @@ -2,7 +2,7 @@ package FIX::Lite;
> >
> >  use vars qw($VERSION @ISA);
> >  use warnings;
> > -use strict;
> > +#use strict;
>
> why did you remove use strict?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/agapoff/FIX-Lite/pull/1/files#r47746105>.
>